examples/chalicelib/models/accounts_redis.py

```diff
@@ -0,0 +1,23 @@
+import datetime as dt
+
+from rom import DateTime, util
+
+from agave.models.redis import RedisModel
+from agave.models.helpers import uuid_field, String
+
+DEFAULT_MISSING_DATE = dt.datetime.utcfromtimestamp(0)
+
+
+class AccountRedis(RedisModel):
+    id = String(
+        default=uuid_field('US'),
+        required=True,
+        unique=True,
+        index=True,
+        keygen=util.IDENTITY,
+    )
+    name = String(required=True, index=True, keygen=util.IDENTITY)
+    user_id = String(required=True, index=True, keygen=util.IDENTITY)
+    secret = String()
+    created_at = DateTime(default=dt.datetime.utcnow, index=True)
+    deactivated_at = DateTime(default=DEFAULT_MISSING_DATE, index=True)
```

Comment:
why is this in `agave`—instead of `authed`? Where else is it used?

Reply:
Este es código solo lo puse para probar los tests con redis.

Reply:
Podemos moverlo a las pruebas entonces?

Reply:
Este codigo deberia estar en la ruta 'examples/chalicelib/models/accounts_redis.py' donde actualmente esta, ya que forma parte de los examples con Redis